syntax = "proto3";
package archive;





/*
CreateAndRegister返回
*/
message CreateAndRegisterResponse {
    /*
    版本Id
    */
    string versionId = 1 ;
    /*
    包文件的md5检验值
    */
    string sign = 2 ;
    /*
    版本的进程及启停、安装、升级脚本, package.conf.yaml
    */
    string conf = 3 ;
}

/*
CreateAndRegisterApi返回
*/
message CreateAndRegisterResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    archive.CreateAndRegisterResponse data = 4 ;
}

